rskj-core/src/main/java/co/rsk/Start.java

```diff
@@ -121,7 +124,9 @@ public void startNode(String[] args) throws Exception {
         logger.info("Running {},  core version: {}-{}", rskSystemProperties.genesisInfo(), rskSystemProperties.projectVersion(), rskSystemProperties.projectVersionModifier());
         BuildInfo.printInfo();
 
+        pendingState.processBest(blockchain.getBestBlock());
```

Comment:
Please add a comment explaining this change.

Reply:
I put this behaviour inside the `.start()` method of the `PendingState`. I added a comment in place though